docs/config/README.md

```diff
@@ -1661,6 +1662,40 @@ The module will be shown if any of the following conditions are met:
 format = "via [🐪 $version]($style) "
 ```
 
+## Perl
+
+The `perl` module shows the currently installed version of Perl.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `Makefile.PL` file
+- The current directory contains a `Build.PL` file
+- The current directory contains a `cpanfile` file
+- The current directory contains a `cpanfile.snapshot` file
+- The current directory contains a `META.json` file
+- The current directory contains a `META.yml` file
+- The current directory contains a `.perl-version` file
+- The current directory contains a `.pl` file
+- The current directory contains a `.pm` file
+- The current directory contains a `.pod` file
+
+### Options
+
+| Variable   | Default      | Description                                           |
+| ---------- | ------------ | ----------------------------------------------------- |
+| `symbol`   | `"🐪 "`      | The symbol used before displaying the version of PHP. |
+| `style`    | `"bold 149"` | The style for the module.                             |
+| `disabled` | `false`      | Disables the `perl` module.                           |
+
+### Example
+
+```toml
+# ~/.config/starship.toml
+
+[perl]
+symbol = "🦪 "
+```
```

Comment:
### Options

| Option     | Default                            | Description                                             |
| ---------- | ---------------------------------- | ------------------------------------------------------- |
| `format`   | `"via [$symbol$version]($style) "` | The format string for the module.                       |
| `symbol`   | `"🐪 "`                            | The symbol used before displaying the version of Perl. |
| `style`    | `"bold 149"`                    | The style for the module.                               |
| `disabled` | `false`                            | Disables the `perl` module.                            |

### Variables

| Variable | Example   | Description                          |
| -------- | --------- | ------------------------------------ |
| version  | `v5.26.1` | The version of `perl`               |
| symbol   |           | Mirrors the value of option `symbol` |
| style\*  |           | Mirrors the value of option `style`  |

\*: This variable can only be used as a part of a style string

### Example

```toml
# ~/.config/starship.toml

[perl]
format = "via [🦪 $version]($style) "
```